viadot/tasks/epicor.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+import xml.etree.ElementTree as ET
+from xml.etree.ElementTree import fromstring
+
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+from typing import Any, Dict, List, Optional
+
+from ..sources import Epicor
+
+
+class EpicorOrdersToDF(Task):
+    def __init__(
+        self,
+        base_url: str,
+        filters_xml: str,
+        credentials: Dict[str, Any] = None,
+        config_key: str = None,
+        start_date_field: str = "BegInvoiceDate",
+        end_date_field: str = "EndInvoiceDate",
+        *args,
+        **kwargs,
+    ) -> pd.DataFrame:
+        """
+        Task for downloading and parsing orders data from Epicor API to a pandas DataFrame.
+
+        Args:
+            name (str): The name of the flow.
+            base_url (str, required): Base url to Epicor Orders.
+            filters_xml (str, required): Filters in form of XML. The date filter is necessary.
+            credentials (Dict[str, Any], optional): Credentials to connect with Epicor Api containing host, port, username and password. Defaults to None.
+            config_key (str, optional): Credential key to dictionary where details are stored. Defauls to None.
+            start_date_field (str, optional) The name of filters filed containing start date. Defaults to "BegInvoiceDate".
+            end_date_field (str, optional) The name of filters filed containing end date. Defaults to "EndInvoiceDate".
+
+        Returns:
+            pd.DataFrame: DataFrame with parsed API output
+        """
+        self.credentials = credentials
+        self.config_key = config_key
+        self.base_url = base_url
+        self.filters_xml = filters_xml
+        self.start_date_field = start_date_field
+        self.end_date_field = end_date_field
+        super().__init__(
+            name="EpicorOrders_to_df",
```

Comment:
that's mixed camel case with snake case; I think we use snake case for task names